shoop/core/pricing/apps.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from shoop.apps import AppConfig
+
+
+class DefaultPricingAppConfig(AppConfig):
```

Comment:
I don't think we need a separate app just to provide `default_pricing`... Just move the provide to core's provides?